### PR TITLE
#5510. Console error while opening Context list

### PR DESCRIPTION
**Pull request by jstrR**
## Description

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**

#5510 

**What is the new behavior?**

returns plain action with type and payload
## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by mbarto** [comment]
@MV88 I have no idea about that action, I never worked on the ManagerMenu, so I don't know exactly what the action is meant to do. @jstrR can you follow the epics / reducers triggered by that action and try to understand what is the behaviour behind the action call (if any)?

**Comment by jstrR** [comment]
@MV88 itemsSelected action is used in other parts of the app, but not here.

**Comment by MV88** [comment]
> @MV88 itemsSelected action is used in other parts of the app, but not here.

ok so i would not remove it we should find a way to dispatch itemSelected too, is that right @mbarto ?

**Comment by jstrR** [comment]
@MV88 dispatches itemsSelected too

File: web/client/plugins/manager/ManagerMenu.jsx [hunk 1]
```diff
@@ -73,7 +72,6 @@ class ManagerMenu extends React.Component {
         }],
         role: "",
         onItemClick: () => {},
-        itemSelected: () => {},
```
Comment (MV88) [inline web/client/plugins/manager/ManagerMenu.jsx:76]: i would leave this one too

File: web/client/plugins/manager/ManagerMenu.jsx [hunk 2]
```diff
@@ -97,7 +95,18 @@ class ManagerMenu extends React.Component {
             .filter(e => this.props.enableContextManager || e.path !== "/context-manager")
             .sort((a, b) => a.position - b.position).map((entry) => {
                 return {
-                    action: (context) => {context.router.history.push(entry.path); return this.props.itemSelected(entry.id); },
+                    action: (context) => {
+                        context.router.history.push(entry.path);
+                        this.props.itemSelected(entry.id);
```
Comment (MV88) [inline web/client/plugins/manager/ManagerMenu.jsx:100]: have you checked that this action is called correctly?

Comment (jstrR) [reply web/client/plugins/manager/ManagerMenu.jsx:101]: this works the same as before my changes

File: web/client/plugins/manager/ManagerMenu.jsx (hunk 1 above)
Comment (jstrR) [reply web/client/plugins/manager/ManagerMenu.jsx]: yeah, i forgot this one

**Comment by MV88** [comment]
please backport to 2020.02.xx

**Pull request merged**